src/main/kotlin/no/nav/syfo/application/api/access/APIConsumerAccessService.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.syfo.application.api.access
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.syfo.application.api.authentication.getConsumerClientId
+import no.nav.syfo.util.configuredJacksonMapper
+
+class APIConsumerAccessService(
+    azureAppPreAuthorizedApps: String
+) {
+    private val preAuthorizedClientList: List<PreAuthorizedClient> = configuredJacksonMapper()
+        .readValue(azureAppPreAuthorizedApps)
+
+    fun validateConsumerApplicationAZP(
+        token: String,
+        authorizedApplicationNameList: List<String>,
+    ) {
+        val consumerClientIdAzp: String = getConsumerClientId(token = token)
```

Comment:
Hva er egentlig `azp`?

Reply:
Fra Azure-doc'en: "The application ID of the client using the token".